soh/src/code/z_play.c

```diff
@@ -273,11 +273,15 @@ void Gameplay_Init(GameState* thisx) {
     u8 tempSetupIndex;
     s32 pad[2];
 
-    if (gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_SKIP_CHILD_STEALTH)) {
+    // Skip Child Stealth when option is enabled, Zelda's Letter isn't obtained and Impa's reward hasn't been received
+    // eventChkInf[4] & 1 = Got Zelda's Letter
+    // eventChkInf[5] & 0x200 = Got Impa's reward
+    // entranceIndex 0x7A, Castle Courtyard - Day
+    // entranceIndex 0x400, Zelda's Courtyard
```

Comment:
might be nice to add "from crawlspace" for the castle courtyard part (no need for a "from" with zelda's courtyard because there's only one way in

Reply:
Sure